examples/templates/jupyter/main.tf

```diff
@@ -0,0 +1,100 @@
+terraform {
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = "0.4.2"
+    }
+  }
+}
+
+variable "cluster_project" {
+  type        = string
+  description = "The Google Cloud project containing the Kubernetes cluster"
+}
+
+variable "cluster_name" {
+  type        = string
+  description = "The cluster name in Google Cloud"
+}
+
+data "google_client_config" "provider" {}
+
+data "google_container_cluster" "my_cluster" {
+  project  = var.cluster_project
+  name     = var.cluster_name
+  location = "us-central1-a"
+}
+
+provider "kubernetes" {
+  host  = "https://${data.google_container_cluster.my_cluster.endpoint}"
+  token = data.google_client_config.provider.access_token
+  cluster_ca_certificate = base64decode(
+    data.google_container_cluster.my_cluster.master_auth[0].cluster_ca_certificate,
+  )
+}
```

Comment:
This should really use Docker, not Kubernetes. If the value we're displaying is Jupyter working in Coder, adding GKE is a pretty massive hurdle.

Reply:
I just copied your template

Reply:
We can make it Docker later but I think this is better than nothing